### PR TITLE
[Release_v2130] Update ReleaseNotes.md

### PR DESCRIPTION
**Pull request by KodiaqQ**
### Changes

- Added v2.13.0 template;

### Reason for changes

- Upcoming release;

### Related tickets

- 151121;

#### For the contributors:
Please add your changes (as the commit to the branch) to the list according to the template and previous notes;
Do not add tests-related notes;
Provide the list of the PRs (for all your notes) in the comment for the discussion;

### CODE REVIEW

**Comment by KodiaqQ** [comment]
@alexsu52, @ljaljushkin, @l-bat, @nikita-savelyevv, @andreyanufr, @andrey-churkin, @daniil-lyakhov, @kshpv, @AlexanderDokuchaev fill the document with your changes for the upcoming release, please.

File: ReleaseNotes.md [hunk 1]
```diff
@@ -1,5 +1,55 @@
...
+
+- Breaking changes:
+  - ...
+- General:
+  - ...
+- Features:
+  - Added a memory monitoring tool for logging the memory a piece of python code or a scpript allocates.
+- Fixes:
+  - Fixed some scenarios of NNCF PyTorch patching interfering with `torch.compile`.
+  - Fixed the quantization of Convolution and LSTMSequence operations in cases where some inputs are part of a ShapeOF subgraph.
```
Comment (andrey-churkin) [inline ReleaseNotes.md:15]: PR https://github.com/openvinotoolkit/nncf/pull/2902

File: ReleaseNotes.md [hunk 2]
```diff
@@ -1,5 +1,57 @@
...
+  - Removes logging of algorithm generated ignored scope.
```
Comment (kshpv) [inline ReleaseNotes.md:16]: https://github.com/openvinotoolkit/nncf/pull/2815

File: ReleaseNotes.md [hunk 3]
```diff
@@ -1,5 +1,57 @@
...
+- Improvements:
+  - Improved progress bar logic during weights compression for more accurate remaining time estimation.
+  - [OpenVINO] Increased hardware fused pattern coverage.
```
Comment (kshpv) [inline ReleaseNotes.md:19]: https://github.com/openvinotoolkit/nncf/pull/2822
https://github.com/openvinotoolkit/nncf/pull/2804

**Review by kshpv — APPROVED** [review]

**Review by AlexanderDokuchaev — APPROVED** [review]

File: ReleaseNotes.md [hunk 4]
```diff
@@ -1,5 +1,61 @@
...
+## New in Release 2.13.0
+
+Post-training Quantization:
+
+- Breaking changes:
+  - ...
+- General:
+  - ...
+- Features:
+  - (OpenVINO) Added LoRA Correction Algorithm to further improve accuracy of int4 compressed models on top of other
```
Comment (ljaljushkin) [inline ReleaseNotes.md:12]: PR https://github.com/openvinotoolkit/nncf/pull/2816

**Review by l-bat — APPROVED** [review]

File: ReleaseNotes.md [hunk 5]
```diff
@@ -1,5 +1,61 @@
...
+  algorithms - AWQ and Scale Estimation. Can be enabled via optional `lora_correction` parameter of
+  `nncf.compress_weights()` API. The algorithm increases compression time and incurs a negligible model size overhead.
+  Refer to [accuracy/footprint trade-off](docs/usage/post_training_compression/weights_compression/Usage.md#accuracyfootprint-trade-off) for different int4 compression methods.
+  - Added a memory monitoring tool for logging the memory a piece of python code or a script allocates.
```
Comment (ljaljushkin) [inline ReleaseNotes.md:16]:
```suggestion
  - Added a memory monitoring tool for logging the memory that allocates a piece of python code or a python script.
```
@nikita-savelyevv looks better, IMO

Comment (nikita-savelyevv) [reply ReleaseNotes.md:16]: I've asked a popular chat bot:

> The first sentence is clearer and grammatically correct:
> 
> "Added a memory monitoring tool for logging the memory a piece of python code or a script allocates."
> 
> This structure makes it clear that the memory is being allocated by the code or script. The second sentence is awkward and less clear, particularly the phrase "that allocates a piece of python code or a python script," which could confuse the subject (what is doing the allocation).

**Review by nikita-savelyevv — APPROVED** [review]

**Review by andreyanufr — APPROVED** [review]

File: ReleaseNotes.md [hunk 6]
```diff
@@ -1,5 +1,64 @@
...
+## New in Release 2.13.0
+
+Post-training Quantization:
+
+- Breaking changes:
+  - ...
+- General:
+  - ...
+- Features:
+  - (OpenVINO) Added the ability to use the Scale Estimation algorithm to calculate quantization parameters in GPTQ for more accurate weight compression. The `gptq=True` and `scale_estimation=True` options can now be used together to enable calculation of quantization parameters using the Scale Estimation algorithm within GPTQ in `nncf.compress_weight`.
```
Comment (alexsu52) [inline ReleaseNotes.md:12]: PR: https://github.com/openvinotoolkit/nncf/pull/2948

**Review by alexsu52 — COMMENTED** [review]
@MaximProshin I have updated release notes from my side.

File: ReleaseNotes.md (hunk 6 above)
Comment (KodiaqQ) [reply ReleaseNotes.md:12]: FYI, the `nncf.compress_weight` method doesn't exist. Only `nncf.compress_weights()`.

**Comment by KodiaqQ** [comment]
@alexsu52, @MaximProshin, review, please.

**Review by alexsu52 — APPROVED** [review]

File: ReleaseNotes.md [hunk 7]
```diff
@@ -1,5 +1,38 @@
 # Release Notes
 
+## New in Release 2.13.0
+
+Post-training Quantization:
+
+- Features:
+  - (OpenVINO) Implemented the use of the Scale Estimation algorithm to calculate quantization parameters within GPTQ for more accurate weight compression. The `gptq` and `scale_estimation` options can now be used together to calculate quantization parameters using the Scale Estimation algorithm within GPTQ in `nncf.compress_weights()`.
```
Comment (MaximProshin) [inline ReleaseNotes.md:8]: Please combine items 1 and 3 to the following:
(OpenVINO) Added support for combining GPTQ with AWQ and Scale Estimation (SE) algorithms in `nncf.compress_weights()` for more accurate weight compression of LLMs. Thus, the following combinations with GPTQ are now supported: AWQ+GPTQ+SE, AWQ+GPTQ, GPTQ+SE, GPTQ.

File: ReleaseNotes.md [hunk 8]
```diff
@@ -1,5 +1,38 @@
...
+  - (OpenVINO) Fixed issue with the FakeConvert duplication for FP8.
+  - Fixed Smooth Quant algorithm issue in case of the incorrect shapes.
+  - Fixed non-deterministic layer-wise scheduling.
+- Improvements:
+  - (OpenVINO) Increased hardware-fused pattern coverage.
+  - Improved progress bar logic during weights compression for more accurate remaining time estimation.
+  - Extended Scale estimation bitness range support for the `nncf.compress_weights()`.
+  - Removed extra logging for the algorithm-generated ignored scope.
+- Tutorials:
+  - [Post-Training Optimization of Flux.1 Model](https://github.com/openvinotoolkit/openvino_notebooks/tree/latest/notebooks/flux.1-image-generation/flux.1-image-generation.ipynb)
```
Comment (MaximProshin) [inline ReleaseNotes.md:24]: Please also add the following tutorials with NNCF support (they were added recently and will be highlighted as part of OV 2024.4):
- https://github.com/openvinotoolkit/openvino_notebooks/tree/latest/notebooks/internvl2
- https://github.com/openvinotoolkit/openvino_notebooks/tree/latest/notebooks/qwen2-audio
- https://github.com/openvinotoolkit/openvino_notebooks/tree/latest/notebooks/nuextract-structure-extraction
- https://github.com/openvinotoolkit/openvino_notebooks/tree/latest/notebooks/minicpm-v-multimodal-chatbot

cc @l-bat @nikita-savelyevv

Comment (KodiaqQ) [reply ReleaseNotes.md:23]: Added

File: ReleaseNotes.md (hunk 7 above)
Comment (KodiaqQ) [reply ReleaseNotes.md]: Updated

**Review by MaximProshin — APPROVED** [review]

**Pull request merged**